README.md

```diff
@@ -55,6 +55,17 @@ For example:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element, { displayMode: true });
 ```
 
+Note that the `throwOnError` option only deals with how TeX related parsing errors are handled.
+Other errors such as passing in an incorrect type for an expression will still throw a TypeError even if the `throwOnError` option is `false`.
+
+For example:
+
+```js
+katex.render("\\unsupportedCommand", element, { throwOnError: true }); // throws ParseError
+katex.render("\\unsupportedCommand", element, { throwOnError: false }); // does not throw ParseError
+katex.render({invalidInputDataType: "yes this is!"}, element, { throwOnError: false }); // throws TypeError
+```
+
```

Comment:
Personally I don't think we need to add three paragraphs here just to discuss that one option. If you think the current wording is easily misread, perhaps add some text like “Other malformed inputs will always throw errors, though” to the description of the option within the bullet list. If you'd like to stress the kind of thing you're fixing here, making it “appropriate errors” in that sentence might express that, but I guess there will be other situations we haven't considered, so I'd not add that. Having useful error messages is useful to the people who encounter them, but in my opinion not worth documenting in the README, since that file should be fairly concise. The example is in my opinion an example of how to use options in general, and we should not have one example for every option.